emcc.py

```diff
@@ -2791,6 +2791,29 @@ def in_directory(root, child):
   return False
 
 
+def parse_string(text):
+  if text[0]=='\'' or text[0]=='"':
```

Comment:
please add spaces before and after `==`

Reply:
also, may as well do `"'"` (avoid escaping a single quote), I think that's more readable